modules/Hoot/ui/conflicts/map.js

```diff
@@ -43,10 +43,18 @@ export default class Map {
         // panning will cause a 'drawn' event to fire and will automatically highlight the nodes
         if ( panTo ) {
             this.panToConflict();
+
+            let feature1Click = d3.selectAll('td.value-col.feature1');
+            feature1Click.on('click', () => this.panToConflict());
+
+            let feature2Click = d3.selectAll('td.value-col.feature2');
+            feature2Click.on('click', () => this.panToConflict());
```

Comment:
I think a better home for this is in `info.js` where the tag table is created.  Also the behavior in Hoot1 includes selecting the entity.

See the code here:
https://github.com/ngageoint/hootenanny-ui/blob/develop/js/hoot/control/conflicts/info/ReviewTable.js#L63-L70

I also notice that after zooming, the extent snaps back to the entire extent of the review instead of just the individual feature.